prefilter:
  imu_covariance:
    accel: [1e-3, 0,    0, 
            0,    1e-3, 0, 
            0,    0,    0]
    angular_velocity: [0, 0, 0, 
                       0, 0, 0, 
                       0, 0, 1e-3]

  wheel_covariance:
    twist: [1e-3, 0,    0, 0, 0, 0,
            0,    1e-3, 0, 0, 0, 0,
            0,    0,    0, 0, 0, 0,
            0,    0,    0, 0, 0, 0,
            0,    0,    0, 0, 0, 0,
            0,    0,    0, 0, 0, 0]

  odometry_covariance:
    twist: [1e-3, 0,    0, 0, 0, 0,
            0,    1e-3, 0, 0, 0, 0,
            0,    0,    0, 0, 0, 0,
            0,    0,    0, 0, 0, 0,
            0,    0,    0, 0, 0, 0,
            0,    0,    0, 0, 0, 0]